nautobot/extras/tests/test_customfields.py

```diff
@@ -283,6 +297,7 @@ def test_get_single_object_without_custom_field_data(self):
             "url_field": None,
             "choice_field": None,
             "multi_choice_field": None,
+            "json_field": None,
         }
         if "dummy_plugin" in settings.PLUGINS:
```

Comment:
Code cleanup item (not required for this PR): all of our tests now always run with the dummy/example plugin enabled, so this `if` check throughout the test isn't really needed any more.